File: Topic Based Problems/01 Fundamentals/2 Intermediate/02 Area of a Parallelogram/Python/CalculateBMI.py
"""
File: CalculateBMI.py
Author: SharifdotG (sharifmdyousuf007@gmail.com)
Brief: Calculate BMI in Python
Version: 1.0
Date: 2023-09-23
"""

weight, height = map(float, input().split())

bmi = weight / (height * height)

print(bmi)

File: Topic Based Problems/01 Fundamentals/2 Intermediate/02 Area of a Parallelogram/Java/CalculateBMI.java
import java.util.Scanner;

public class CalculateBMI {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        double weight = scanner.nextDouble();
        double height = scanner.nextDouble();

        double bmi = weight / (height * height);

        System.out.println(bmi);
        
        scanner.close();
    }
}
